pwa/src/components/deleteModal/DeleteModal.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from "react";
+import "./deletemodal.css";
+import { Modal } from "@conductionnl/nl-design-system";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faTrash } from "@fortawesome/free-solid-svg-icons";
+
+interface DeleteModalProps {
+  handleDelete: any;
+  handleId: any;
+  optionalLength?: any;
```

Comment:
Very nice catch, that we (might or probably should) want to give the user feedback if he's deleting more than just the resource that he's clicking on.

However, I think we could probably do this in a more generic way (see other comments regarding the `optionalLength`).

Note: this shouldn't be `:any`.

Note 2: you're actually checking if it's `=== 1`, this would break on a string.